### PR TITLE
Persistence graphical tool improvements

### PR DESCRIPTION
**Pull request by VincentRouvreau**
- Fix issue #113 (replace `is not` with `!=`)
- Fix issue #109  (replace palette with a more visible one)

### CODE REVIEW

File: src/common/include/gudhi/reader_utils.h [hunk 1]
```diff
@@ -305,8 +305,12 @@ void read_persistence_intervals_and_dimension(std::string const& filename, Outpu
     std::string line;
     getline(in, line);
     if (line.length() != 0 && line[0] != '#') {
-      double numbers[4];
+      double numbers[4] = {0.};
```
Comment (mglisse) [inline src/common/include/gudhi/reader_utils.h]: Is it because of one of gcc's idiotic warnings?

Comment (VincentRouvreau) [reply src/common/include/gudhi/reader_utils.h]: I add a weird bug and could not find an explanation. I tried to set the double array before filling it, but it did not help.
I did not remove it, but I can do it.

Comment (mglisse) [reply src/common/include/gudhi/reader_utils.h]: If it does not help for anything, then yes, I'd rather not add the initialization.
Is the weird bug still there?

Comment (VincentRouvreau) [reply src/common/include/gudhi/reader_utils.h]: Yes it is still there, even with the master branch. It does not seem to come from the development done in this branch

Comment (VincentRouvreau) [reply src/common/include/gudhi/reader_utils.h]: I removed the initialization on 57c1f0f4b36b934c8fc54c8691f77b9b6d0eff9d

Comment (mglisse) [reply src/common/include/gudhi/reader_utils.h]: Ça ne colle pas bien avec le DEBUG_TRACES en dessous. Il faudrait soit que le code dans DEBUG_TRACES ne lise que les n premiers éléments, soit qu'on vire ce code, soit qu'on initialise (avec 4 qNaN par exemple ?).

Comment (mglisse) [reply src/common/include/gudhi/reader_utils.h]: > Yes it is still there, even with the master branch. It does not seem to come from the development done in this branch

ok, then file an issue and let's ignore it for this PR.

Comment (VincentRouvreau) [reply src/common/include/gudhi/reader_utils.h]: Ok.
I reworked traces on 1e3b4f6c56409dde933341701575dc105ede12db
The issue is tracked by #111

**Pull request merged**